packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -3513,12 +3561,17 @@ export function pushStartInstance(
     case 'span':
     case 'svg':
     case 'path':
-    case 'a':
     case 'g':
     case 'p':
     case 'li':
       // Fast track very common tags
       break;
+    case 'a':
+      if (enableFilterEmptyStringAttributesDOM) {
+        return pushStartAnchor(target, props);
+      } else {
+        break;
+      }
```

Comment:
If I put it back up I have to make an additional check that we're dealing with an `a` tag so that e.g. `div` wouldn't fall into `pushStartAnchor`. 

As far as I can tell, I can only put it before the common group or after but not in between without having an additional check. But then maybe I'm missing some neat tricks with `switch` statements that I haven't thought of.

Reply:
> just add the extra break before it?

That was the important bit I missed.